pkg/cmd/opts/kustomize_test.go

```diff
@@ -0,0 +1,33 @@
+package opts
+
+import (
+	"github.com/jenkins-x/jx/pkg/versionstream"
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+var versionTests = []struct {
+	currentVersion string
+	stableVersion  *versionstream.StableVersion
+	supported      bool
+}{
+	{"3.5.1", &versionstream.StableVersion{Version: "3.5.4", UpperLimit: "3.6.0"}, false},
+	{"3.5.4", &versionstream.StableVersion{Version: "3.5.4", UpperLimit: "3.6.0"}, true},
+	{"3.6.0", &versionstream.StableVersion{Version: "3.5.4", UpperLimit: "3.6.0"}, false},
+	{"{3.5.4  2020-01-17T14:28:58+00:00  }", &versionstream.StableVersion{Version: "3.5.4", UpperLimit: "3.6.0"}, true},
```

Comment:
This test and the one below make not much sense anymore, since we do the version extraction now in _kustomize_cli#Version_ directly.

See comment on that function.

Reply:
Shall I move this to kustomize_cli_test?